scripts/spack/configs/versions.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# This file lists the package versions for axom's dependencies
+#
+
+packages:
+  adiak:
+    require: 
+      - spec: "@0.4.0"
+  caliper:
+    require:
+      - spec: "@2.12.1"
+  camp:
+    require:
+      - spec: "@2025.03.0"
+  conduit:
+    require:
+      - spec: "@0.9.3"
```

Comment:
Just curious but why 0.9.3 over 0.9.4?

Reply:
Only b/c I didn't update conduit in this PR -- it's using the same commit, I just added a version to it's clearer. 

It might be cool to have a separate follow-up PR to go to 0.9.4 to see what needs to change for a new version.